tests/test_s3.py

```diff
@@ -3626,3 +3627,41 @@ def run_policy(policy):
         lifecycle = client.get_bucket_lifecycle_configuration(Bucket=bname)
         self.assertEqual(len(lifecycle["Rules"]), 1)
         self.assertEqual(lifecycle["Rules"][0]["ID"], lifecycle_id2)
+
+
+@terraform('aws_s3_encryption_audit')
+def test_s3_encryption_audit(test, aws_s3_encryption_audit):
+    test.patch(s3.S3, "executor_factory", MainThreadExecutor)
+    test.patch(s3.S3, "S3_AUGMENT_TABLE", [])
```

Comment:
this is patching the class, it should be patching the module.. ie its currently having no effect, there's a bunch of recordings here from the augment still.